packages/@aws-cdk/cx-api/test/util.ts

```diff
@@ -0,0 +1,23 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * rm -rf reimplementation, don't want to depend on an NPM package for this
```

Comment:
What's wrong with using an npm package for tests?

Reply:
Let's see. I can either have this 10-line function or

```
└─┬ rimraf@3.0.2
  └─┬ glob@7.1.6
    ├── fs.realpath@1.0.0
    ├─┬ inflight@1.0.6
    │ ├── once@1.4.0 deduped
    │ └── wrappy@1.0.2
    ├── inherits@2.0.4
    ├─┬ minimatch@3.0.4
    │ └─┬ brace-expansion@1.1.11
    │   ├── balanced-match@1.0.0
    │   └── concat-map@0.0.1
    ├─┬ once@1.4.0
    │ └── wrappy@1.0.2 deduped
    └── path-is-absolute@1.0.1
```